### ISSUE TITLE
trusty build fail due to kazoo module

### ISSUE THREAD

**Issue by linuxmaniac** [kamailio/kamailio#418]
```
make[1]: Entering directory `/tmp/buildd/kamailio-4.4.0~dev6+0~20151124215712.399+trusty'
mkdir -p /tmp/buildd/kamailio-4.4.0~dev6+0~20151124215712.399+trusty/debian/kamailio-kazoo-modules/usr/lib/x86_64-linux-gnu/kamailio/modules
CC (gcc) [M kazoo.so]		const.o
CC (gcc) [M kazoo.so]		kazoo.o
CC (gcc) [M kazoo.so]		kz_amqp.o
kz_amqp.c: In function 'kz_amqp_exchange_declare':
kz_amqp.c:57:2: error: incompatible type for argument 7 of 'amqp_exchange_declare'
  return amqp_exchange_declare(state, channel, exchange, type, passive, durable, 0, 0, arguments);
  ^
In file included from /usr/include/amqp.h:720:0,
                 from kz_amqp.c:4:
/usr/include/amqp_framing.h:610:61: note: expected 'amqp_table_t' but argument is of type 'int'
 AMQP_PUBLIC_FUNCTION amqp_exchange_declare_ok_t * AMQP_CALL amqp_exchange_declare(amqp_connection_state_t state, amqp_channel_t channel, amqp_bytes_t exchange, amqp_bytes_t type, amqp_boolean_t passive, amqp_boolean_t durable, amqp_table_t arguments);
                                                             ^
kz_amqp.c:57:2: error: too many arguments to function 'amqp_exchange_declare'
  return amqp_exchange_declare(state, channel, exchange, type, passive, durable, 0, 0, arguments);
  ^
In file included from /usr/include/amqp.h:720:0,
                 from kz_amqp.c:4:
/usr/include/amqp_framing.h:610:61: note: declared here
 AMQP_PUBLIC_FUNCTION amqp_exchange_declare_ok_t * AMQP_CALL amqp_exchange_declare(amqp_connection_state_t state, amqp_channel_t channel, amqp_bytes_t exchange, amqp_bytes_t type, amqp_boolean_t passive, amqp_boolean_t durable, amqp_table_t arguments);
                                                             ^
make[2]: *** [kz_amqp.o] Error 1
make[1]: *** [install-modules] Error 1
make[1]: Leaving directory `/tmp/buildd/kamailio-4.4.0~dev6+0~20151124215712.399+trusty'
```

git version 81be39ba472bb3c1a8569c7158c6415aac512be7

Full build log https://kamailio.sipwise.com/job/kamailiodev-nightly-binaries/architecture=amd64,distribution=trusty,label=slave/397/consoleText

**Comment by linuxmaniac** [comment]
This is related to #411 and 29aaeb5

- trusty: librabbitmq1 0.4.1-1
- jessie: librabbitmq1 0.5.2-2
- sid ( and soon stretch): librabbitmq4 0.7.1-1

Not sure I @lazedo wants to support trusty for next 4.4 version

**Comment by linuxmaniac** [comment]
Fixed by aef56c53f0
